crates/bevy_ui/src/render/mod.rs

```diff
@@ -348,11 +361,10 @@ pub fn extract_text_uinodes(
                 let atlas_size = Some(atlas.size);
 
                 // NOTE: Should match `bevy_text::text2d::extract_text2d_sprite`
```

Comment:
Is this comment (and the corresponding one in `text2d.rs`) still accurate? The two chunks of code seems to have diverged quite a bit.

Reply:
It's not really good either way. If there is duplication it should probably be abstracted away.

I think delete it, but leave the comment in Text2d.rs as outside the scope of this PR.

Reply:
Looked at text2d.rs some more and fixed some issues (including removing the other comment) in #6741